libp2p/switch.nim

```diff
@@ -250,6 +245,21 @@ proc stop*(s: Switch) {.async.} =
 
   trace "Switch stopped"
 
+proc updatePeerInfo*(
```

Comment:
Is this being called anywhere?

Reply:
Not ATM (only required in a test), but this is in preparation to the integration of push identify in the switch.
Not done in this PR because this will change a few things too unrelated

Reply:
I've moved to PeerStore, now it's used both by the regular upgrade flow and the push identify